tests/integration/testdata/buildcmd/terraform/image_based_lambda_functions_local_backend/main.tf

```diff
@@ -109,4 +109,23 @@ resource "null_resource" "sam_metadata_aws_lambda_function_l2_function" {
         docker_build_args = ""
         docker_tag = "latest"
     }
+}
+
+## Serverless TF
+module "docker_image" {
+  source = "git::https://github.com/moelasmar/terraform-aws-lambda.git//modules/docker-build?ref=master_sam_cli_integration_null_resource_solution"
```

Comment:
we probably want to change this?

Reply:
this should be updated after our PR on serverless TF module got merged and released to terraform registry